src/lib.rs

```diff
@@ -202,6 +202,31 @@ impl<'a> MailHeader<'a> {
 
         result
     }
+
+    /// Get the raw, unparsed value of the header value.
+    ///
+    /// # Examples
+    /// ```
+    ///     use mailparse::parse_header;
+    ///     let (parsed, _) = parse_header(b"Subject: =?iso-8859-1?Q?=A1Hola,_se=F1or!?=").unwrap();
```

Comment:
For this example please modify the key text to be a little more "interesting", like `SuBJect : =?iso...` so that it's clearer from the example what kinds of case/whitespace normalization is done on the key.